cmd/frontend/graphqlbackend/oobmigrations.go

```diff
@@ -32,7 +34,10 @@ func (r *schemaResolver) OutOfBandMigrationByID(ctx context.Context, id graphql.
 }
 
 // OutOfBandMigrations resolves all registered single out-of-band migrations.
-func (r *schemaResolver) OutOfBandMigrations(ctx context.Context) ([]*outOfBandMigrationResolver, error) {
+// If filterDeprecatedBeforeFirstVersion is set to true, migrations deprecated before the init version of sourcegraph will be filtered out.
+func (r *schemaResolver) OutOfBandMigrations(ctx context.Context, args *struct {
+	FilterDeprecatedBeforeFirstVersion *bool
```

Comment:
nit: not a native speaker, but i think "exclude" captures the intention better, as "filter" could be seen as "filter only for ...". Alternatively, "FilterOutDeprecatedBeforeFirstVersion" is equally better.
```suggestion
	ExcludeDeprecatedBeforeFirstVersion *bool
```

Reply:
I agree this is a better title 👍🏼